Codeinwp/neve
# Improve TC visibility

### Summary
- Added the card in neve dashboard
- Changed the Notification react component to accept just a text + a link, until now it was only working only with plugins and themes updates

### Will affect visual aspect of the product
NO

### Test instructions
- The card should only appear if the White label is deactivated. Test if this is true.
- The link on the card should say "Discover Templates Cloud" if you don't have the Agency plan with an URL to Neve pricing and should say "Learn how to use Template Cloud" with a link to [documentation](https://docs.themeisle.com/article/1091-starter-sites-available-for-import#my-library)

After you test this, you need to have a look at [this PR too](https://github.com/Codeinwp/templates-patterns-collection/pull/79).


Closes Codeinwp/neve-pro-addon#1286.
